ipa_moveit_config/srdf/ur_macro.srdf.xacro

```diff
@@ -40,7 +40,7 @@
     </group_state>
     <!--END EFFECTOR - Purpose - Represent information about an end effector.-->
     <!--VIRTUAL JOINT - Purpose - this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="${prefix}fixed_base" type="fixed" parent_frame="world" child_link="${prefix}base_link" />
+    <joint name="${prefix}fixed_base" type="fixed" parent_frame="world" child_link="${prefix}base_link" />
```

Comment:
`virtual_joints` are currently not really supported, this silences the warning for now